packages/lesswrong/components/revisions/RevisionSelect.tsx

```diff
@@ -47,7 +47,7 @@ const styles = (theme: ThemeType): JssStyles => ({
 
 const RevisionSelect = ({ revisions, getRevisionUrl, onPairSelected, loadMoreProps, classes, count, totalCount }: {
   revisions: Array<RevisionMetadataWithChangeMetrics>,
-  getRevisionUrl: (rev: RevisionMetadata) => React.ReactNode,
+  getRevisionUrl: (rev: RevisionMetadata) => string,
```

Comment:
This was used in 2 places and in both places it was return a string.  The type signature needs to be changed with the types now function for `Link`, because `<Link to={...}>` does not allow `React.ReactNode` as a `to` value.